include/ensmallen_bits/moead/moead.hpp

```diff
@@ -0,0 +1,322 @@
+/**
+ * @file moead.hpp
+ * @authors Utkarsh Rai, Nanubala Gnana Sai
+ *
```

Comment:
```suggestion
 * @author Utkarsh Rai
 * @author Nanubala Gnana Sai
 *
```

That's the format we used in other files.